test/Website/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM microsoft/dotnet:1.0-runtime
+
```

Comment:
Do you think this is necessary in this case, given that it is just being used to run integration tests. We are not actually going to be publishing this docker image anywhere.